webapp/src/index.css

```diff
@@ -1,13 +1,88 @@
+/* bring in normalize.css styles */
+@import-normalize;
+
+:root {
+  --color-warning: #ff3a6a;
+  --color-submit: #4b52ee;
+  --color-online: #3fc6a2;
+  --color-offline: #dfc04d;
+  --color-light: #ffffff;
+  --color-background: #191b1f;
+  --color-dark: #262934;
+  --color-game: linear-gradient(
+    269.73deg,
+    #4b52ee 40.15%,
+    #fb4e78 78.32%,
+    #3fc6a2 99.73%
+  );
+}
+
+html {
+  box-sizing: border-box;
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: inherit;
+}
+
+html,
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
-    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+  padding: 0;
+}
+
+.text-style-0-bold {
+  font-size: 3rem;
+  font-family: 'DM Sans', sans-serif;
+  font-weight: 700;
+}
+
+.text-style-1-bold {
+  font-size: 3rem;
+  font-family: 'DM Sans', sans-serif;
+  font-weight: 700;
+}
+
+.text-style-2-bold {
+  font-size: 1.5rem;
+  font-family: 'DM Sans', sans-serif;
+  font-weight: 700;
+}
+
+.text-style-3-bold {
+  font-size: 1.25rem;
+  font-family: 'DM Sans', sans-serif;
+  font-weight: 700;
+}
+
+.text-style-4-regular {
+  font-size: 1rem;
+  font-family: 'DM Sans', sans-serif;
+  font-weight: 400;
+}
+
+.text-style-4-medium {
```

Comment:
I don't get it: we created the landing to test colors and fonts (and sizes), but we are only using one combination in the page?

Reply:
The `Landing` page is a placeholder we could use as an example of how to structure the pages. I didn't want to add too much content since we will replace it with the actual content and styles when we start implementing the pages.